src/analyzer.py

```diff
@@ -142,6 +145,19 @@ def noun_phrase(input_text):
     return n_phrase_lst
 
 
+
+def concatenate(responses_df):
```

Comment:
Tokens with stop words stripped are already provided by `tokenize()`. See if you can reuse that.

Reply:
Upon further analysis, it looks like we don't need to remove stop words during concatenate because the word cloud method automatically removes stop words from the string of all words that is inputted. The wordcloud package also needs a concatenated string of all words rather than a list of tokens, so tokenize() wouldn't work well as a replacement.